test/openBCISimulator-test.js

```diff
@@ -137,7 +140,9 @@ describe('openBCISimulator', function () {
       simulator._startStream();
     });
     it('should return a sync set packet with accel', function (done) {
-      var simulator = new openBCISimulator.OpenBCISimulator(k.OBCISimulatorPortName);
+      var simulator = new openBCISimulator.OpenBCISimulator(k.OBCISimulatorPortName, {
+        fragmentation: k.OBCISimulatorFragmentationNone
```

Comment:
Should not need to change existing tests with a minor version change. please remove this change.